internal/pkg/errors/template_errors.go

```diff
@@ -220,11 +220,7 @@ func (p *PackTemplateError) fixupPackContextable() {
 		return
 	}
 
-	atRE, err := regexp.Compile(`^.*"` + p.Filename + `" at <(.+)>$`)
-	if err != nil {
-		// there's a janky filename that's breaking the regex.
-		return
-	}
+	atRE := regexp.MustCompile(`^executing .* at <(.+)>$`)
```

Comment:
Since the regex seems static now you can declare it once at the top of the file so only need to compile once, but no biggie since it's a CLI tool

Reply:
Good call. Moved them up